integration-tests/test/test_propose.py

```diff
@@ -1,24 +1,58 @@
 import os
 from random import Random
-
-from docker.client import DockerClient
-
-from .common import (
-    CommandLineOptions,
+from .rnode import (
+    extract_validator_stake_from_deploy_cost_str,
+    parse_show_block_output
 )
-from .conftest import testing_context
-from .rnode import docker_network_with_started_bootstrap
-from .wait import (
-    wait_for_approved_block_received_handler_state,
+from .rnode import (
+    Node,
 )
 
 
-def test_propose(command_line_options: CommandLineOptions, random_generator: Random, docker_client: DockerClient) -> None:
-    with testing_context(command_line_options, random_generator, docker_client) as context:
-        with docker_network_with_started_bootstrap(context=context) as bootstrap_node:
-            wait_for_approved_block_received_handler_state(context, bootstrap_node)
-            relative_paths = bootstrap_node.shell_out('sh', '-c', 'ls /opt/docker/examples/*.rho').splitlines()
-            relative_path = random_generator.choice(relative_paths)
-            full_path = os.path.join('/opt/docker/examples', relative_path)
-            bootstrap_node.deploy(full_path)
-            bootstrap_node.propose()
```

Comment:
So we no longer need propose() ?

Reply:
Why not? That's an easy way to get the newly deployed block hash.

Reply:
So I guess I'm confused because the below code doesn't contain propose()

```python
def test_propose(started_standalone_bootstrap_node: Node, random_generator: Random) -> None:
    relative_paths = started_standalone_bootstrap_node.shell_out('sh', '-c', 'ls /opt/docker/examples/*.rho').splitlines()
    relative_path = random_generator.choice(relative_paths)
    full_path = os.path.join('/opt/docker/examples', relative_path)
    started_standalone_bootstrap_node.deploy(full_path)
```